wagtailio/static/sass/components/_logos.scss

```diff
@@ -0,0 +1,38 @@
+.logos {
+    @include sf-spacing();
+
+    &__heading {
+        margin-bottom: 20px;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            margin-bottom: 40px;
+            grid-column: 2 / span 3;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 4;
+        }
+    }
+
+    &__list {
+        display: flex;
+        flex-wrap: wrap;
+        list-style: none;
+        grid-column: 2 / span 2;
+        gap: 20px;
```

Comment:
All looks good @chris-lawton! I think that maybe we could centre the list in mobile view, it leaves a little too much space for my taste on the right, but feel free to ignore.

Reply:
I agree, think the white space on the right looks odd. Let 's centre them : )